### ISSUE TITLE
[BUG] check-unused-code doesn't exclude files from config

### ISSUE THREAD

**Issue by furaiev** [dart-code-checker/dart-code-metrics#653]
* **Dart code metrics version:** 4.10.0-dev.1
* **Dart sdk version:**  2.15.0 (stable)

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```yaml
analyzer:
  exclude:
    - "**/*.g.dart"
    - "**/*.gr.dart"
    - "**/*.freezed.dart"
    - "**/generated/**.dart"
    - "**/*.gform.dart"
    - "ios/**"
  strong-mode:
    implicit-casts: false
    implicit-dynamic: false
  errors:
    missing_required_param: warning
    missing_return: error
  plugins:
    - dart_code_metrics

dart_code_metrics:
  anti-patterns:
    - long-method
    - long-parameter-list
  metrics:
    cyclomatic-complexity: 27
    source-lines-of-code: 50
    number-of-parameters: 4
    maximum-nesting-level: 5
  metrics-exclude:
    - test/**
    - e2e/**
    - ios/**
  rules:
    - always-remove-listener
    - avoid-non-null-assertion
    - avoid-returning-widgets
    - avoid-unnecessary-setstate
    - avoid-unused-parameters
    - avoid-wrapping-in-padding
    - double-literal-format
    - newline-before-return
    - no-boolean-literal-compare
    - prefer-match-file-name:
        exclude:
          - test/**
    - prefer-single-widget-per-file
    - prefer-trailing-comma
    - no-equal-then-else
    - binary-expression-operand-order
```

</details>

**What did you do? Please include the source code example causing the issue.**

```sh
flutter pub run dart_code_metrics:metrics check-unused-code lib
```

**What did you expect to happen?**
generated files from the `exclude` section are excluded from the analysis

**What actually happened?**
![image](https://user-images.githubusercontent.com/33546696/149941483-eb8e00fa-4ac3-42ec-8bb6-cbd7f6c845ff.png)

**Comment by incendial** [comment]
Should be fixed here: https://github.com/dart-code-checker/dart-code-metrics/pull/654

**Comment by incendial** [comment]
@furaiev fix is available in 4.10.0-dev.2, please check it out!

**Comment by furaiev** [comment]
@incendial checked, it is fixed, thanks a lot!
